nntrainer/models/neuralnet.cpp

```diff
@@ -994,7 +997,10 @@ sharedConstTensors NeuralNetwork::incremental_inference(
   /** @todo: deallocate tensor after incremental inference **/
   /** Clear the set inputs and labels */
   model_graph.setInputsLabels({}, {});
-
+  auto fsu_enable = std::get<props::Fsu>(model_flex_props);
+  if (fsu_enable) {
+    model_graph.inActive(0);
```

Comment:
In the previous code, variable f was used in the inActive. 
However, in the modified code, only the value 0 is used.
What is the reason behind this change?

Reply:
In the previous logic, elements were inactivated according to the execution order. However, given the current operation method of FSU, it is more efficient to handle this once per inference rather than processing it layer by layer. This change was made for efficiency.

The part you mentioned is a remnant of the previous method where it was handled order by order. 
To prevent confusion, I will correct it immediately.

Reply:
Signature Change Done! 👍